rush.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
   "npmVersion": "8.19.4",
```

Comment:
Lets see if this works, if not we might have to bump this as well (see the 1DS change)